config/locales/en.yml

```diff
@@ -605,6 +605,8 @@ en:
   constraint_type_codes:
     article4: Article 4 direction area
     article4_caz: Central activities zones
+    articleFour: Article 4 direction area
+    articleFour_caz: Central activities zones
```

Comment:
Do we also want to run any migration to change it on the [database](https://github.com/unboxed/bops/blob/main/db/schema.rb#L227)?

Reply:
Not yet - it's not in production yet so we'd get new requests with `article4` in them.